src/Modules/MatchTrackerModule/Services/MatchTracker.cs

```diff
@@ -114,6 +114,37 @@ await events.RaiseAsync(MatchTrackerEvent.StateTracked,
         }
     }
 
+    public async Task TrackMapChangeAsync(MapEventArgs mapArgs)
+    {
+        if (!IsTracking)
+        {
+            return;
+        }
+        
+        await VerifyTracker();
+
+        IMatchState state;
+        
+        state = new MapMatchState
+        {
+            Status =
+                MatchStatus.Unknown, // FIXME Either we need to assume the match is running when the server map is changed (bad) or we override the status to unknown (also bad).
```

Comment:
A match will always be running whem the tracker is enabled, so setting to running is the correct one.

Reply:
This is now changed.